### ISSUE TITLE
Voila-vuetify will be broken with Voila 0.5.0

### ISSUE THREAD

**Issue by martinRenou** [voila-dashboards/voila-vuetify#64]
## Description

This is a warning that we are making some deep (and breaking) changes in Voila 0.5.0:

- Voila becomes a Lab-remix https://github.com/voila-dashboards/voila/pull/846. This means that you won't have access to the kernel API anymore here: https://github.com/voila-dashboards/voila-vuetify/blob/master/share/jupyter/nbconvert/templates/vuetify-base/util.js#L152
- Voila will use JupyterLab mime renderers https://github.com/voila-dashboards/voila/pull/1249 This might break your assumptions concerning outputs rendering (I'm not completely sure though, because it seems you are not inheriting from the Voila templates?)

**Comment by jtpio** [comment]
For reference trying to use the template with `voici` (which is based on Voila 0.5.0) leads to the following error:

![image](https://user-images.githubusercontent.com/591645/231750541-54144ce3-a627-48ac-a585-820031dd224d.png)

This was noticed in https://github.com/voila-dashboards/voici/pull/60.

**Comment by maartenbreddels** [comment]
Thanks for the heads up!
What is the expected release date of 0.5 do you think?

**Comment by martinRenou** [comment]
Hopefully next month for JupyterCon

**Comment by jhgoebbert** [comment]
I just stumbled over this Voila-vuetify template which seems to be compatible with voila 0.5.0: 
https://pypi.org/project/voila-vuetify-tuwien/

**Comment by dfguerrerom** [comment]
> I just stumbled over this Voila-vuetify template which seems to be compatible with voila 0.5.0: https://pypi.org/project/voila-vuetify-tuwien/

I think they have removed that package... @jhgoebbert do you know if that's the same as this one https://pypi.org/project/voila-vuetify-template-tuwien/#files ? this doesn't work with `voila>5`

**Comment by jhgoebbert** [comment]
Yes, that seems to be the case: https://colab.tuwien.ac.at/pages/viewpage.action?pageId=183404699#Voilà-VuetifyDefaultTemplate

**Comment by dfguerrerom** [comment]
> Yes, that seems to be the case: [https://colab.tuwien.ac.at/pages/viewpage.action?pageId=183404699#Voilà-VuetifyDefaultTemplate](https://colab.tuwien.ac.at/pages/viewpage.action?pageId=183404699#Voil%C3%A0-VuetifyDefaultTemplate)

yes, it uses `voila-0.4.3` :/

**Comment by maartenbreddels** [comment]
I'm curious, is there a particular feature you need from voila 0.5?

**Comment by martinRenou** [comment]
> I'm curious, is there a particular feature you need from voila 0.5?

Maybe Voici support :D

**Comment by dfguerrerom** [comment]
> I'm curious, is there a particular feature you need from voila 0.5?

Actually no... we are doing an upgrade in our system and we are avoiding all the unnecessary pins in our libraries, but yes, `voila~0.4` can meet our current needs.

**Comment by dfguerrerom** [comment]
@maartenbreddels, @martinRenou , I have created a new Voila template based on voila-vuetify, which you can find here: https://github.com/dfguerrerom/voila-sepal-ui. It works with voila<0.5. After spending some time working with this template, I've gained a good understanding of how the templates function. 
I am now planning to make it compatible with voila==0.5, and I could also submit a PR to this repository to update it accordingly. 
As you mentioned, the Lab-remix changes how we access the Kernel API. Could you give me a hint about where to look to access this API in the new version? It's important because I want to be able to access the `_handleMessage` and `_update_status` functions.
